source/guides/stacks/mers.rst

```diff
@@ -81,13 +110,25 @@ Components of the |MERS| include:
 * `OpenVINO™ toolkit
   <https://01.org/openvinotoolkit>`_ for inference.
 
-* `FFmpeg* <https://www.ffmpeg.org>`_ with `Scalable Video Technology (SVT)
-  <https://01.org/svt>`_ plugins for encoding, decoding, and transcoding.
+* `FFmpeg* <https://www.ffmpeg.org>`_ with plugins for:
 
-* `GStreamer* <https://gstreamer.freedesktop.org/>`_  with `Scalable Video
-  Technology (SVT) <https://01.org/svt>`_ and `OpenVINO™ toolkit
-  <https://01.org/openvinotoolkit>`_ plugins for analytics.
+   - `Scalable Video Technology (SVT)
+     <https://01.org/svt>`_
 
+* `GStreamer* <https://gstreamer.freedesktop.org/>`_  with plugins for:
+
+   - `Scalable Video
+     Technology (SVT) <https://01.org/svt>`_
+   - `OpenVINO™ toolkit
+     <https://01.org/openvinotoolkit>`_
+   - `VAAPI <https://github.com/GStreamer/gstreamer-vaapi>`_
+
+* `Intel® Media SDK <https://github.com/Intel-Media-SDK/MediaSDK>`_ 
+
+.. note::
+
+   The |MERS| is validated against Gen 11 Intel® Processor Graphics and newer.
```

Comment:
I think "Gen 11" should be spelled out and lower case - its not part of the official name. "11th generation Intel® Processor Graphics"